circle.yml

```diff
@@ -21,6 +28,14 @@ test:
     - npm test # Run the package and docs test suite
     - npm run build:package # Run the release process
     - npm run build:website # Build Jekyll based docs website
+    - jekyll serve -w:
+       background: true
+    # start tunnel
+    - ./screener-tunnel-client -apikey="$SCREENER_API_KEY" -host="ci.server:4000":
+        background: true
+    - sleep 20
+    # run tests
+    - ./screener-ci.sh $SCREENER_API_KEY $SCREENER_GROUP_ID $CIRCLE_BUILD_NUM
```

Comment:
yep, the build fails if any changes are detected. Someone will have to validate those changes and kick off the build again once those changes are accepted.